src/main/java/org/molgenis/vcf/decisiontree/loader/model/ConfigBoolClause.java

```diff
@@ -0,0 +1,76 @@
+package org.molgenis.vcf.decisiontree.loader.model;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+public class ConfigBoolClause {
+
+  String id;
+  List<ConfigBoolQuery> queries;
+  ConfigNodeOutcome outcomeTrue;
+  ConfigClauseOperator operator = null;
+
+  public ConfigBoolClause() {
+    this.id = UUID.randomUUID().toString();
```

Comment:
Why does a BoolClause need an id and a BoolQuery doesn't?

Reply:
Since it has an outcome attachted to it, it needs a second pass when creating the tree, an ID is needed here to match the object from the first pass with the config object